plugin/ui.browserscope.js

```diff
@@ -1044,22 +1044,25 @@
     // the frame window of the charts
     me.chartWindow = iwin;
 
-    // the element the charts are inserted into
-    me.container = query('#bs-chart', idoc)[0];
-
-    // Browserscope's UA div is inserted before an element with the id of "bs-ua-script"
-    loadScript('https://www.browserscope.org/ua?o=js', me.container).id = 'bs-ua-script';
-
-    // the "autoload" string is created following the guide at
-    // https://developers.google.com/loader/?hl=en#auto-loading
-    loadScript('https://www.google.com/jsapi?autoload=' + encodeURIComponent('{' +
-      'modules:[{' +
-        'name:"visualization",' +
-        'version:1,' +
-        'packages:["corechart","table"],' +
-        'callback:ui.browserscope.load' +
-      '}]' +
-    '}'), idoc);
+    // ensure that content in iframe is processed by executing on next tick
+    setTimeout(function() {
+      // the element the charts are inserted into
+      me.container = query('#bs-chart', idoc)[0];
```

Comment:
Yeah, it should be inside. Previously `me.container` was undefined as the iframe hadn't updated yet. Placing a timeout of 1 should get the code to run in the next tick, and the iframe should be updated by then.